### PR TITLE
edge: update websocket test

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <long0dai@foxmail.com>

**What type of PR is this?**
 /kind test

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: edge/pkg/edgehub/clients/wsclient/websocket_test.go [hunk 1]
```diff
@@ -125,6 +120,11 @@ func TestNewWebSocketClient(t *testing.T) {
 			newTestWebSocketClient("normal", "/tmp/edge.crt", "/tmp/edge.key"),
 		},
 	}
+
+	if err := newTestServer(); err != nil {
```
Comment (fisherxu) [inline edge/pkg/edgehub/clients/wsclient/websocket_test.go:124]: Will other tests use this fake server?

Comment (daixiang0) [reply edge/pkg/edgehub/clients/wsclient/websocket_test.go:124]: Only this.

**Comment by daixiang0** [comment]
Ref https://github.com/kubeedge/kubeedge/issues/2243

**Pull request merged**